src/lib/seam/components/ClimateSettingScheduleDetails/ClimateSettingScheduleDetails.tsx

```diff
@@ -0,0 +1,97 @@
+import classNames from 'classnames'
+import { useState } from 'react'
+
+import { formatDateAndTime } from 'lib/dates.js'
+import { useClimateSettingSchedule } from 'lib/seam/thermostats/climate-setting-schedules/use-climate-setting-schedule.js'
+import { AccordionRow } from 'lib/ui/layout/AccordionRow.js'
+import { ContentHeader } from 'lib/ui/layout/ContentHeader.js'
+import { DetailRow } from 'lib/ui/layout/DetailRow.js'
+import { DetailSection } from 'lib/ui/layout/DetailSection.js'
+import { DetailSectionGroup } from 'lib/ui/layout/DetailSectionGroup.js'
+import Switch from 'lib/ui/Switch/Switch.js'
+import { ClimateSettingStatus } from 'lib/ui/thermostat/ClimateSettingStatus.js'
+
+import { ClimateSettingScheduleCard } from './ClimateSettingScheduleCard.js'
+
+export interface ClimateSettingScheduleDetailsProps {
+  climateSettingScheduleId: string
+  onBack?: () => void
+  className?: string
+}
+
+export function ClimateSettingScheduleDetails({
+  climateSettingScheduleId,
+  onBack,
+  className,
+}: ClimateSettingScheduleDetailsProps): JSX.Element | null {
+  const { climateSettingSchedule } = useClimateSettingSchedule(
+    climateSettingScheduleId
+  )
+
+  const [allowManualOverride, setAllowManualOverride] = useState(false)
+
+  if (climateSettingSchedule == null) {
+    return null
+  }
+
+  return (
+    <div
+      className={classNames('seam-climate-setting-schedule-details', className)}
+    >
+      <ContentHeader title={t.climateSettingSchedule} onBack={onBack} />
+      <div className='seam-climate-setting-schedule-details-content'>
+        <ClimateSettingScheduleCard
+          climateSettingScheduleId={climateSettingScheduleId}
+        />
+        <div className='seam-default-setting-message-container'>
+          <span className='seam-default-setting-message'>
+            {t.defaultSettingMessagePart1}{' '}
+            <span className='seam-default-setting-text'>
+              {t.defaultSetting}
+            </span>{' '}
+            {t.defaultSettingMessagePart2}
+          </span>
+        </div>
+        <DetailSectionGroup>
+          <DetailSection>
+            <AccordionRow label={t.startEndTime} />
+            <AccordionRow
+              label={t.climateSetting}
+              rightCollapsedContent={
+                <ClimateSettingStatus
+                  climateSetting={climateSettingSchedule}
+                  iconPlacement='right'
+                />
+              }
+            />
+            <DetailRow label={t.allowManualOverride}>
+              <Switch
+                checked={allowManualOverride}
+                onChange={setAllowManualOverride}
```

Comment:
This would need to be hooked up before release.

Reply:
Would you be okay with releasing if the switch wasn't present or was disabled? I just wanted the toggle to do something. Might take a bit to make this functional.

Reply:
As discussed IRL, remove the switch for now, but display the label so that the component is read-only.

In the same vein, should I not use accordion rows here? They're currently empty.

Reply:
Ah good call. Yea let's remove the empty expansions. Can we add some issues for what's missing here?

Reply:
I'll make sure we have issues in the project board

Reply:
<img width="1030" alt="Screenshot 2023-08-31 at 3 35 10 PM" src="https://github.com/seamapi/react/assets/11382084/f41f8b7f-3673-4b26-9cfb-e970041047d8">

(Seemed better than a commit hash)